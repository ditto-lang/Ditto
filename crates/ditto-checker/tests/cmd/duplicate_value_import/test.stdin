module Test exports (..)

import Foo (five, five)

fives = [five, five, five]
