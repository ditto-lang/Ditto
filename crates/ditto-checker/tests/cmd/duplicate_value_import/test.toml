bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_value_import.ditto"]
fs.sandbox = true
