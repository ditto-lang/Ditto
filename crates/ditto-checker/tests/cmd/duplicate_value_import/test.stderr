
  ⚠ duplicate value import
   ╭─[duplicate_value_import.ditto:2:1]
 2 │ 
 3 │ import Foo (five, five);
   ·             ──┬─  ──┬─
   ·               │     ╰── already imported
   ·               ╰── previously imported here
 4 │ 
   ╰────

